.github/workflows/ci.yml

```
@@ -16,11 +16,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - run: rustup toolchain install stable --profile minimal
-
       - uses: Swatinem/rust-cache@v2
         with:
-          # bump this to invalidate build/dependency cache
+          # Bump this to manually invalidate the build/dependency cache
+          # It is also keyed on the job, rustc version, Cargo.lock, and other
+          # values explained here: <https://github.com/Swatinem/rust-cache#cache-details>
```

```suggestion
          # values explained here: https://github.com/Swatinem/rust-cache#cache-details
```
Is it automatically highlighted by some tool when `<>` are used?

---

I know some terminals and markup languages want urls bracketed, I think I got that habit from there. Not a requirement here.